recipes/scala/src/main/scala/com/adobe/platform/ml/ScoringDataSaver.scala

```diff
@@ -60,11 +62,13 @@ class ScoringDataSaver extends DataSaver {
     scored_df = scored_df.withColumn("_id", lit("empty"))
     scored_df = scored_df.withColumn("eventType", lit("empty"))
 
-    scored_df.select(tenantId, "_id", "eventType", "timestamp").write.format("com.adobe.platform.dataset")
-      .option(DataSetOptions.orgId, orgId)
-      .option(DataSetOptions.serviceToken, serviceToken)
-      .option(DataSetOptions.userToken, userToken)
-      .option(DataSetOptions.serviceApiKey, apiKey)
+    scored_df.select(tenantId, "_id", "eventType", "timestamp").write.format(PLATFORM_SDK_PQS_PACKAGE)
+      .option("user-token", userToken)
```

Comment:
Once we add back in the DataSetOptions, we need to use the constants here instead of these strings.